src/coprocessor/dag/rpn_expr/impl_op.rs

```diff
@@ -53,10 +54,128 @@ impl RpnFnLogicalOr {
     }
 }
 
+#[derive(Clone, Debug, RpnFunction)]
+#[rpn_function(args = 1)]
+pub struct RpnFnIsNull<T: Evaluable> {
+    _phantom: std::marker::PhantomData<T>,
+}
+
+impl<T: Evaluable> Copy for RpnFnIsNull<T> {}
+
+impl<T: Evaluable> RpnFnIsNull<T> {
+    #[inline]
+    pub fn new() -> Self {
+        Self {
+            _phantom: std::marker::PhantomData,
+        }
+    }
+
+    #[inline]
+    fn call(
+        _ctx: &mut EvalContext,
+        _payload: RpnFnCallPayload<'_>,
+        arg: &Option<T>,
+    ) -> Result<Option<i64>> {
+        Ok(Some(arg.is_none() as i64))
+    }
+}
+
+#[derive(Clone, Copy, Debug, RpnFunction)]
+#[rpn_function(args = 1)]
+pub struct RpnFnIntIsTrue;
+
+impl RpnFnIntIsTrue {
+    #[inline]
+    fn call(
+        _ctx: &mut EvalContext,
+        _payload: RpnFnCallPayload<'_>,
+        arg: &Option<Int>,
+    ) -> Result<Option<i64>> {
+        Ok(*arg)
```

Comment:
it should be `v != 0i64`?

Reply:
The behavior of `is true` for Int is the same as evaluating the int as mysql bool, so I just return the arg.
Maybe I should add a comment or use the evident way?

Reply:
@zz-jason PTAL. Please help verify in what scenario this function will be used. I tried some expressions in TiDB and according to EXPLAIN this function does not occur 🤔

Reply:
See: https://github.com/pingcap/tidb/blob/d9e02b76ddb7a99d6740728edb58612d5b617b0f/expression/builtin_op.go#L475

Reply:
I think it's better to keep consistent with TiDB.